outbound/state/outbound_state.py

```diff
@@ -0,0 +1,132 @@
+import utilities.integration_adaptors_logger as log
+from edifact.outgoing.models.interchange import InterchangeHeader, InterchangeTrailer
+from edifact.outgoing.models.message import MessageHeader, MessageTrailer, ReferenceTransactionNumber, \
+    ReferenceTransactionType
+from utilities.message_utilities import get_uuid
+
+from persistence import persistence_adaptor as pa
+
+DATA = 'DATA'
+OPERATION_ID = 'OPERATION_ID'
+TRANSACTION_ID = 'TRANSACTION_ID'
+TRANSACTION_TIMESTAMP = 'TRANSACTION_TIMESTAMP'
+TRANSACTION_TYPE = 'TRANSACTION_TYPE'
+SIS_SEQUENCE = 'SIS_SEQUENCE'
+SMS_SEQUENCES = 'SMS_SEQUENCES'
+SENDER = 'SENDER'
+RECIPIENT = 'RECIPIENT'
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+
+
+class OutboundState(object):
+    """A local copy of an instance of an outbound state from the state store"""
+
+    def __init__(self, persistence_store: pa.PersistenceAdaptor, store_data: dict):
+        """
+        Given
+        :param persistence_store:
+        :param store_data:
+        """
+        if persistence_store is None:
```

Comment:
Consider creating the instance of persistence_store here instead of passing it in as a constructor argument. That way the caller doesn't need any knowledge of persistence.

Reply:
The table name could be static property of OutboundState also. Nothing else should need to know the table name.

Reply:
done